tools/samba/illumina_suite/macro.xml

```diff
@@ -0,0 +1,30 @@
+<macros>
+    <token name="@TOOL_VERSION_FIGARO@">1.1.2</token>
+    <token name="@TOOL_VERSION_SWARM@">3.1.5</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <xml name="figaro_requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION_FIGARO@">figaro</requirement>
+            <yield/>
+        </requirements>
+    </xml>
+    <xml name="swarm_requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION_SWARM@">swarm</requirement>
+            <yield/>
+        </requirements>
+    </xml>
+    <xml name="figaro_edam">
```

Comment:
This is something I'll need on to find the right edam ontologies for figaro (I tried to find on Bio tools but it's not obvious to me which are the ontologies if they are there)